lib/fathead/caniuse/output.txt

```diff
@@ -1,390 +1,1176 @@
-iframe-seamless	A										<section class="prog__container"><p>The seamless attribute makes an iframe's contents actually part of a page, and adopts the styles from its hosting page. The attribute has been [removed from the current specification](https://github.com/whatwg/html/issues/331). </p><ul><li>IE 11 &#10007;</li><li>Firefox 53</li><li>Chrome 58 &#10007;</li><li>Safari TP &#10007;</li><li>iOS Safari 10-10.1 &#10007;</li><li>Android Browser 53 &#10007;</li></ul><span class="prog__sub">Notes:</span><ul><li>Chrome 20-26 had partial support behind a flag, though this was later removed.</li></ul></section>	http://caniuse.com/iframe-seamless
-css-counters	A										<section class="prog__container"><p>Method of controlling number values in generated content, using the `counter-reset` and `counter-increment` properties.</p><ul><li>IE 8+ &#10003;</li><li>Firefox 2+ &#10003;</li><li>Chrome 4+ &#10003;</li><li>Safari 3.1+ &#10003;</li><li>iOS Safari 3.2+ &#10003;</li><li>Android Browser 2.1+ &#10003;</li></ul></section>	http://caniuse.com/css-counters
-css-page-break	A										<section class="prog__container"><p>Properties to control the way elements are broken across (printed) pages.</p><ul><li>IE 5.5+ partial</li><li>Firefox 2+ partial</li><li>Chrome 4+ partial</li><li>Safari 3.1+ partial</li><li>iOS Safari 3.2+ partial</li><li>Android Browser 2.1+ partial</li></ul><span class="prog__sub">Notes:</span><ul><li>Not all mobile browsers offer print support; support listed for these is based on browser engine capability.</li></ul></section>	http://caniuse.com/css-page-break
+svg filters	A										<section class="prog__container"><p>Method of using photoshop-like effects on SVG objects including blurring and color manipulation.</p><ul><li>IE 10+ &#10003;</li><li>Firefox 3+ &#10003;</li><li>Chrome 8+ &#10003;</li><li>Safari 6+ &#10003;</li><li>iOS Safari 6.0-6.1+ &#10003;</li><li>Android Browser 4.4.3-4.4.4+ &#10003;</li></ul></section>	http://caniuse.com/svg-filters
+svg-filters	R	svg filters
+speech recognition api	A										<section class="prog__container"><p>Method to provide speech input in a web browser.</p><ul><li>IE 11 &#10007;</li><li>Firefox 53 &#10007;</li><li>Chrome 25+ partial (-webkit)</li><li>Safari TP &#10007;</li><li>iOS Safari 10-10.1 &#10007;</li><li>Android Browser 53 &#10007;</li></ul></section>	http://caniuse.com/speech-recognition
+speech recognition	R	speech recognition api
+speech-recognition	R	speech recognition api
+battery status api	A										<section class="prog__container"><p>Method to provide information about the battery status of the hosting device.</p><ul><li>IE 11 &#10007;</li><li>Firefox 43+ &#10003;</li><li>Chrome 38+ &#10003;</li><li>Safari TP &#10007;</li><li>iOS Safari 10-10.1 &#10007;</li><li>Android Browser 53+ &#10003;</li></ul></section>	http://caniuse.com/battery-status
+battery status	R	battery status api
+battery-status	R	battery status api
```

Comment:
Whoops, I commented on the wrong line, see here for example:

https://github.com/duckduckgo/zeroclickinfo-fathead/pull/683/files#diff-59e7fd3cba8ae3361171e752b24431f8R607